### PR TITLE
NCDA Report - Infrastructure, Environment & Wash pane

### PR DESCRIPTION
**Pull request by anvmn**
#521

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1257

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -7860,6 +7873,33 @@ translationSet trans =
...
+                    , kinyarwanda = Nothing
+                    }
+
+                HasCleanWater ->
+                    { english = "Household has clean water"
+                    , kinyarwanda = Nothing
+                    }
+
+                HasHandwashingFacility ->
+                    { english = "Household has handwashing facility  "
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: Extra space at the end here.

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Minor comment. Looks good.

**Pull request merged**